modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/utils/SW360ReleaseAdapterUtils.java

```diff
@@ -78,9 +78,9 @@ private static void setCoordinates(SW360Release release, Artifact artifact) {
 
     private static void setFinalLicense(SW360Release release, Artifact artifact) {
         LicenseInformation licenseInformation = ArtifactLicenseUtils.getFinalLicenses(artifact);
-
-        release.setFinalLicense(licenseInformation.evaluate());
-
+        if(!licenseInformation.evaluate().equals("")) {
```

Comment:
Multiple calls of `licenseInformation.evaluate()`, alternative with `Optional` could be:

```suggestion
         Optional.of(licenseInformation.evaluate())
             .filter(e -> !"".equals(e))
             .ifPresent(e -> release.setFinalLicense(e));
```